mat64/list_test.go

```diff
@@ -383,7 +384,8 @@ func testTwoInput(c *check.C,
 		&basicMatrix{},
 		&basicVectorer{},
 		&basicSymmetric{},
-		&basicTriangular{},
```

Comment:
Because we can accidentally write functions that assume Uplo is set properly, even though it is not always. See for example https://github.com/gonum/matrix/issues/171 .